packages/@aws-cdk/aws-ecs/test/fargate/integ.capacity-provider.ts

```diff
@@ -0,0 +1,26 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
```

Comment:
Both added. Please note as we yet to support the `AutoScaling Group capacity provider` https://github.com/aws/aws-cdk/pull/9192, chances are users may have a cluster with `FARGATE` and `FARGATE_SPOT` as its default capacity provider strategy and they are allow to create `Ec2Service` as well.  We have two cases in this scenaro:

1. create `Ec2Service` with no `capacityProviderStrategy` specified. In this case the `laucnhType: EC2` will be explicitly specified. Everything works as expected.
2. create `Ec2Service` with `capacityProviderStrategy` specified. As capacity provider strategy and launch type can't both specified, we will honor the `capacityProviderStrategy` instead for `Ec2Service` and it could create Fargate tasks as we only offer fargate capacity providers at this moment.